--- servicio-cursos/pom.xml2026-05-11 16:35:00.000000000 -0400
+++ servicio-cursos/pom.xml2026-05-11 16:35:00.000000000 -0400
@@ -62,8 +62,11 @@
 </dependencies>
 
 <dependencyManagement>
 <dependencies>
-<type>pom</type>
-<scope>import</scope>
-</dependency>
+<dependency>
+<groupId>org.springframework.cloud</groupId>
+<artifactId>spring-cloud-dependencies</artifactId>
+<version>${spring-cloud.version}</version>
+<type>pom</type>
+<scope>import</scope>
+</dependency>
 </dependencies>
